Competition2018/src/main/java/competition/subsystems/offboard/OffboardInterfaceSubsystem.java

```diff
@@ -112,6 +116,25 @@ public void sendSetCurrentCommand(int commandId) {
         return queueContents;
     }
     
+    private void sendScoringPlacement() {
+        //Sequence - Near, Scale, Far
+        //Left - 1, Right - 0
+        ArrayList<Integer> sequence = new ArrayList<Integer>();
+        for(MatchData.GameFeature feature: MatchData.GameFeature.values()) {
```

Comment:
Can you move this logic to the `pack...` method? This is internal logic which I think fits better over there.

Reply:
!